service/Abstractions/Search/SearchClientConfig.cs

```diff
@@ -40,6 +41,42 @@ public class SearchClientConfig
     /// </summary>
     public string EmptyAnswer { get; set; } = "INFO NOT FOUND";
 
+    //
```

Comment:
nit: pls use the usual documentation format (triple forward slashes, xmldocs, etc), and add valid ranges also in the comments